src/common/verbose.cpp

```diff
@@ -514,6 +514,7 @@ std::string md2fmt_str(
             break;
         case format_kind::wino:
         case format_kind::rnn_packed:
+        case format_kind::cublaslt_blocked:
```

Comment:
would it make sense to print something meaningful here? (e.g. the tags documented by cublasLt)

Reply:
@mgouicem, so we make it semi-opaque? :)

Reply:
This was mostly a suggestion for debugging purpose. It could potentially help debugging from verbose output to know what the internal primitive did with internal layouts.
We do the same with zero_padding for example. It is not a public API, yet we print some info relative to zero padding in verbose log to help with debugging.

Reply:
Added a meaningful output. If you decide to change to no output for format, let me know and we can revert.

Reply:
I thinks it's a reasonable approach.